### DISCUSSION TITLE
Welcome to DelphiVCL4Python Discussions!

### DISCUSSION THREAD

**Discussion by jimmckeeth** [Embarcadero/DelphiVCL4Python#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by danieleteti** [comment]
Hi, I'm Daniele Teti (CEO & CTO @ bittimeprofessionals).
I love Delphi and Python since 1999, so this is THE community  ;-)

**Comment by Lucioric2000** [comment]
I am Lucio Montero, a freelance Python developer at Upwork (https://www.upwork.com/freelancers/luciomontero). I am a beta tester for the DelphiVCL and DelphiFMX modules.

**Comment by lmbelo** [comment]
Hi, I'm Lucas. 
I'm a participant and contributor. 
I'm here to get suggestions, answer questions, share experiences and have fun.

**Comment by Priyatham10** [comment]
Hello Everyone,
I feel overjoyed to mention that I'm one of the contributors to this project. I'd love people to please check out this project and I bet you'll fall in love with it as I did.
